cms/djangoapps/contentstore/management/commands/tests/test_create_course.py

```diff
@@ -46,9 +46,6 @@ class TestCreateCourse(ModuleStoreTestCase):
     Unit tests for creating a course in either old mongo or split mongo via command line
     """
 
-    def setUp(self):
-        super(TestCreateCourse, self).setUp(create_user=True)
```

Comment:
I thought this was an edx-lint violation...do we want to encourage a different pattern now?